src/views/static/Status.vue

```diff
@@ -122,7 +119,7 @@ export default {
       })
     },
     fetchLatestVersionForService (service) {
-      const githubLink = this.statuses[service].githubLink || ('https://api.github.com/repos/AEGEE/' + service + '/contents/package.json')
```

Comment:
Was `this.statuses[service].githubLink` only used for Core?

Reply:
yep, because of the `oms-core-elixir`/`oms-core-js` difference